.github/workflows/icu4c.yml

```diff
@@ -910,9 +924,15 @@ jobs:
       - name: Zip
         run: |
           cd icu4c/source/icuexportdata
-          zip -r ../../../icuexportdata_tag-goes-here.zip .
+          zip -r ../../../icuexportdata_${{ inputs.gitReleaseTag }}.zip .
```

Comment:
The suffix in the filename after the `_` underscore needs to replace all `/` slash characters with a `-` dash. The release tag names for ICU4X specific preview-releases  for the purposes of icuexportdata will have slashes in the name.

Reply:
I tried to keep this to a minimum, to get the exact same results as before.

The trouble is that everything else uses the ICU version (with 75.1 for Java, 75_1 for C).

But in fact this does not use the version, it uses the GitHub tag.
Which includes the `release-` part.
See the url here: https://github.com/unicode-org/icu/releases/tag/release-74-2 for example.

To really unify this I would need to also change the Makefile for icu4c (OK, the stuff used to generate the Makefile :-)

TLDR: I didn't want to shake that boat now, so close to RC.
I'm fine to do it between PR and release (if acceptable), or for the next version.